nmap.aliases.bash

```diff
@@ -0,0 +1,15 @@
+#Niil78
+#2019
```

Comment:
No need to have the year here either...

Reply:
Sorry. It is my first contribution.
I really made the aliases for me. But I wanted to upload them here.